packages/zenn-cli/src/server/commands/new-article.ts

```diff
@@ -78,8 +80,9 @@ export const exec: CliExecFn = (argv) => {
       `type: "${type}" # tech: 技術記事 / idea: アイデア`,
       'topics: []',
       `published: ${published}`,
+      publicationName !== '' ? `publication-name: ${publicationName}` : null,
```

Comment:
タイトルと同じようにサニタイズしてもらえると、より安全になると思います。

```suggestion
      publicationName !== '' ? `publication-name: ${publicationName.replace(/"/g, '\\"')}` : null,
```

Reply:
入力サニタイズしていたんですね、見逃していました。
コミット取り込みました、ありがとうございます:pray: https://github.com/zenn-dev/zenn-editor/pull/373/commits/af0b62a23b56e24a1923bd15d2630c4f6d14a863